src/configuration/cloudformation.coffee

```diff
@@ -48,9 +49,23 @@ apiConfig = async (dir, globals) ->
   yield preprocessors.api mungedConfig
 
 renderAPI = async (dir, globals) ->
+  H = require "handlebars"
   mungedConfig = yield apiConfig dir, globals
-  template = yield read resolve skyMixinsPath, "api.yaml"
-  yaml _render template, mungedConfig
+  # TODO: factor this out into something saner
```

Comment:
This is where we're wandering into Emerald Robot land. Not sure if it's worth the intermediate step of cleaning this up within Panda Sky or if we're ready to just make the leap?